script.100fm/resources/settings.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<settings>
+    <category>
+        <setting label="10000" type="number" id="channel" default="0" visible="false"/>
```

Comment:
Sorry, I didnt spot everything during first review.... I am fine with this for now, but would be cool if you could take care of this for newer versions.

You can avoid that re-submit by force-pushing to the same branch btw... that saves you some time, and us too, since we get a diff then and dont have to go through all the code again.